### ISSUE TITLE
Interpolation problems in `NotoTraditionalNushu[wght].ttf`

### ISSUE THREAD

**Issue by behdad** [notofonts/nushu#8]
Hello!

This is an automatically-generated report about possible interpolation problems in `NotoTraditionalNushu[wght].ttf`, as found in the Google Fonts catalog.

The particular version of the font that was tested was [NotoTraditionalNushu-v2.003/NotoTraditionalNushu-v2.003.zip](https://github.com/notofonts/nushu/releases/download/NotoTraditionalNushu-v2.003/NotoTraditionalNushu-v2.003.zip).

To download a PDF version of this report with helpful visuals of the problems, click [here](https://github.com/behdad/google-fonts-ofl-interpolatable-results/raw/8ce3e0535f17cd069e91df45e252462bf74a2cc4/reports/NotoTraditionalNushu%5Bwght%5D.ttf.pdf); Or to view it on the GitHub website, click [here](https://github.com/behdad/google-fonts-ofl-interpolatable-results/blob/8ce3e0535f17cd069e91df45e252462bf74a2cc4/reports/NotoTraditionalNushu%5Bwght%5D.ttf.pdf).

The report follows:
```
Glyph u1B1CE was not compatible:
  Masters: '', 'wght=300.0':
    Contour 2 start point differs: 0 in '', 5 in 'wght=300.0'; reversed: False
Glyph u1B2F1 was not compatible:
  Masters: '', 'wght=300.0':
    Contour order differs: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12] in '', [0, 1, 2, 9, 10, 11, 12, 3, 4, 5, 6, 7, 8] in 'wght=300.0'
```

This report was generated using the `fonttools varLib.interpolatable` tool.  We understand that sometimes the tool generates false-positives.  Particularly for more complicated font designs.  If you did not find this report useful, please apologize and ignore & close it.

To give feedback about this report, please file an issue or open a discussion at [fonttools](https://github.com/fonttools/fonttools).

**Comment by simoncozens** [comment]
@behdad: I think these are false positives. (In both cases there are rotations, which the tool struggles with.)

**Comment by behdad** [comment]
> @behdad: I think these are false positives. (In both cases there are rotations, which the tool struggles with.)

The first one looks legit to me.

**Comment by behdad** [comment]
Let me reproduce with hb-view.

**Comment by behdad** [comment]
I confirmed with hb-view both are real issues. Try at `wght=350`.

**Comment by simoncozens** [comment]
Already fixed in 836ef3c, that's why it looks OK... :-)
